jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -214,18 +214,18 @@ const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = (
 
   const [curation, curationDispatch] = useReducer(sortingCurationReducer, defaultSortingCuration)
   sorting.curation = curation
+
+  const [selection, selectionDispatch] = useReducer(sortingSelectionReducer, defaultSortingSelection)
```

Comment:
Here's where we maintain the selection state for the jupyter widget. In the future we'll want to attach this to a python-defined context where we can share the selection state between different jupyter widgets.